Sources/Datadog/RUM/RUMMonitor/Scopes/RUMViewScope.swift

```diff
@@ -244,6 +247,22 @@ internal class RUMViewScope: RUMScope, RUMContextProvider {
         )
     }
 
+    private func addDiscreteUserAction(on command: RUMAddUserActionCommand) {
+        userActionScope = createDiscreteUserActionScope(on: command)
+    }
+
+    private func sendDiscreteCustomUserAction(on command: RUMAddUserActionCommand) {
+        let customActionScope = createDiscreteUserActionScope(on: command)
+        _ = customActionScope.process(
+            command: RUMStopUserActionCommand(
+                                    time: command.time,
```

Comment:
I would say yes, but on Android it will be `1ns` (not possible to have 0), so I have to check this in web UI to confirm that `0` works fine, otherwise we use `1ns`. Anyway, the idea is to have instant action, without any duration (hence no dependencies possible).